daily-test/variables.yaml

```diff
@@ -0,0 +1,33 @@
+project:
+  id: 'eq-daily-performance-test'
+  folder_id: ''
+  billing_account: ''
+  region: 'europe-west2'
+  label: 'sandbox-eq'
+
+k8s:
+  node_machine_type: 'n1-standard-1'
+  timeout: '60m'
```

Comment:
This should be under `benchmark` as this is the timeout for the Job.

Reply:
This is the timeout for the pipeline?

Reply:
Pipelines have other timeouts, this is really the job timeout. Under k8s it seems like its related to the cluster somehow.